src/main/java/ch/njol/skript/expressions/ExprTime.java

```diff
@@ -76,56 +64,65 @@ public Class<?>[] acceptChange(final ChangeMode mode) {
 		switch (mode) {
 			case ADD:
 			case REMOVE:
-				return CollectionUtils.array(Timespan.class);
+				// allow time to avoid conversion to timespan, which causes all sorts of headaches
+				return CollectionUtils.array(Time.class, Timespan.class);
 			case SET:
 				return CollectionUtils.array(Time.class, Timeperiod.class);
-			case DELETE:
-			case REMOVE_ALL:
-			case RESET:
```

Comment:
Was there any reason for the removal of reset?

Reply:
reset was never implemented, and also which value should be used for it?

Reply:
ahh I'm completely blind, thinking about it again it likely doesn't make sense, since my first though is back to `0:00`